starters/vue3-xstate-css/README.md

```diff
@@ -1,68 +1,204 @@
-# vue3-xstate-css
+# Vue 3, XState, and CSS Starter Kit
 
-This template should help get you started developing with Vue 3 in Vite.
+This starter kit uses [Vue 3](https://vuejs.org/), [XState](https://xstate.js.org/) for state management, and pure CSS for styling.
 
-## Recommended IDE Setup
+**How to use this starter kit**
 
-[VSCode](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (and disable Vetur) + [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin).
+```sh
+npm create @this-dot/starter -- --kit vue3-xstate-css
+```
 
-## Type Support for `.vue` Imports in TS
+<sub>(use any of `npm` / `yarn` / `pnpm`)</sub>
 
-TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) to make the TypeScript language service aware of `.vue` types.
+If you'd prefer manual installation, you can follow this process:
 
-If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has also implemented a [Take Over Mode](https://github.com/johnsoncodehk/volar/discussions/471#discussioncomment-1361669) that is more performant. You can enable it by the following steps:
+> Note: The command below clones the entire repository, including other starter kits.
```

Comment:
Personally, I feel offering this seconds way of cloning the entire repository is bit of a waste of the valuable above the fold space.  The people who would want to clone the entire repository, probably already know how to, and so, I would remove this and replace it with a link to a wiki entry for it